l10n_fr_hr_check_ssnid/i18n/l10n_fr_hr_check_ssnid.pot

```diff
@@ -0,0 +1,40 @@
+# Translation of Odoo Server.
+# This file contains the translation of the following modules:
+# 	* l10n_fr_hr_check_ssnid
+#
+msgid ""
+msgstr ""
+"Project-Id-Version: Odoo Server 14.0\n"
+"Report-Msgid-Bugs-To: \n"
+"Last-Translator: \n"
+"Language-Team: \n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: \n"
+"Plural-Forms: \n"
+
+#. module: l10n_fr_hr_check_ssnid
+#: model:ir.model.fields,field_description:l10n_fr_hr_check_ssnid.field_hr_employee__display_name
+msgid "Display Name"
+msgstr ""
+
+#. module: l10n_fr_hr_check_ssnid
+#: model:ir.model,name:l10n_fr_hr_check_ssnid.model_hr_employee
+msgid "Employee"
+msgstr ""
+
+#. module: l10n_fr_hr_check_ssnid
+#: model:ir.model.fields,field_description:l10n_fr_hr_check_ssnid.field_hr_employee__id
+msgid "ID"
+msgstr ""
+
+#. module: l10n_fr_hr_check_ssnid
+#: model:ir.model.fields,field_description:l10n_fr_hr_check_ssnid.field_hr_employee____last_update
+msgid "Last Modified on"
+msgstr ""
+
+#. module: l10n_fr_hr_check_ssnid
+#: code:addons/l10n_fr_hr_check_ssnid/models/hr_employee.py:0
+#, python-format
+msgid "The French Social Security Number '%s' is invalid. (%s)"
```

Comment:
Could you update the fr.po file according to the change '%s' -> '%(ssnid)s' ?

thanks !

Reply:
Yes she can but don't you think this the weblate job ?

Reply:
When porting a module, updating the Po according to the change you introduce is a good thing. (One single step)
Specially for l10n_fr modules that have only one traduction by design.

Reply:
done